extensions/data-workspace/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "data-workspace",
   "displayName": "Data Workspace",
   "description": "Additional common functionality for database projects",
-  "version": "0.6.0",
+  "version": "0.6.1",
   "publisher": "Microsoft",
   "preview": true,
-  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
+	"license": "SEE LICENSE IN LICENSE.txt",
```

Comment:
Given that we previously had a link to the raw license here, why not do the same thing with the new license location?